cmk/special_agents/agent_proxmox_ve.py

```diff
@@ -526,7 +526,7 @@ def agent_proxmox_ve_main(args: Args) -> None:
     snapshot_data = {}
 
     for node in data["nodes"]:
-        node_timezones[node["node"]] = node["time"]["timezone"]
+        node_timezones[node["node"]] = node["time"].get("timezone", None)
```

Comment:
Unfortunately this only a partial fix. We use the timezone info later to convert the timestamps of backups to utc. The better solution is to not add the node into the `node_timezones` dict. 

```python
if (tz:=node['time'].get('timezone')) is not None:
    node_time[node['node']] = tz
```

Reply:
Agreed, pushed change and deployed it in my setup again.